lib/trigger/send-scheduled-email.ts

```diff
@@ -39,14 +39,31 @@ export const sendDataroomTrialExpiredEmailTask = task({
       });
       logger.info("Email sent", { to: payload.to, teamId: payload.teamId });
 
-      await prisma.team.update({
+      const updatedTeam = await prisma.team.update({
         where: {
           id: payload.teamId,
         },
         data: {
           plan: team.plan.replace("+drtrial", ""),
         },
       });
+
+      const isPaid = ["pro", "business", "datarooms", "datarooms-plus"].includes(updatedTeam.plan);
+      if (!isPaid) {
+        await prisma.team.update({
+          where: {
+            id: payload.teamId,
+          },
+          data: {
+            brand: {
+              delete: true
+            }
+          }
+        })
```

Comment:
better to directly delete the brand from the `brand` model 

await prisma.brand.deleteMany({ where teamId })